presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -475,7 +484,23 @@ public CounterStat getHighMemorySplitSource()
 
                 results.add(new HivePartitionMetadata(hivePartition, Optional.of(partition), partitionSchemaDifference.build(), encryptionInformation));
             }
-
+            StringBuilder warningMessage = new StringBuilder(format("Table '%s' has '%s' out of '%s' partitions unreadable: ", tableName, unreadablePartitionsSkipped, partitionBatch.size()));
```

Comment:
Some single quotes are not needed:
`"Table '%s' has %s out of %s partitions unreadable: "`